src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml

```diff
@@ -198,6 +198,22 @@
 			Canvas.ZIndex="0"
 			EmptyTextType="{x:Bind ParentShellPageInstance.FilesystemViewModel.EmptyTextType, Mode=OneWay}" />
 
+		<Border>
```

Comment:
The border control is extra, can you remove it?

Reply:
That should be it.